lib/nokogiri/xml/node.rb

```diff
@@ -192,16 +198,57 @@ def decorate!
 
       # :section: Manipulating Document Structure
 
-      ###
-      # Add +node_or_tags+ as a child of this Node.
-      #
-      # +node_or_tags+ can be a Nokogiri::XML::Node, a ::DocumentFragment, a ::NodeSet, or a String
-      # containing markup.
-      #
-      # Returns the reparented node (if +node_or_tags+ is a Node), or NodeSet (if +node_or_tags+ is
-      # a DocumentFragment, NodeSet, or String).
-      #
-      # Also see related method +<<+.
+      # :call-seq:
+      #   add_child(object) -> node or nodeset
```

Comment:
When possible, I'd like to use the proper constant name when we reference classes.

```suggestion
      #   add_child(object) -> Node or NodeSet
```

Reply:
Again, different from what's seen in Ruby core and standard libraries.  And again, I'll do as you say.